benchmarks/benchmarks/analysis/distances.py

```diff
@@ -128,3 +132,32 @@ def time_between(self, num_atoms):
                           A=self.ag1,
                           B=self.ag2,
                           distance=15.0)
+
+    def time_calc_bonds(self,num_atoms):
+        np.random.seed(17809)
+        self.coords_1 = np.random.random_sample((num_atoms, 3)).astype(np.float32)
+        np.random.seed(9008716)
+        self.coords_2 = np.random.random_sample((num_atoms, 3)).astype(np.float32)
+       
+        MDAnalysis.lib.distances.calc_bonds(self.coords_1,self.coords_2)
```

Comment:
Can you change the import in this file to the idiomatic `import MDAnalysis as mda` and make the corresponding changes to the function calls?